src/Mvc/Mvc.Core/src/Formatters/SystemTextJsonOutputFormatter.cs

```diff
@@ -40,7 +40,10 @@ internal static SystemTextJsonOutputFormatter CreateFormatter(JsonOptions jsonOp
             if (jsonSerializerOptions.Encoder is null)
             {
                 // If the user hasn't explicitly configured the encoder, use the less strict encoder that does not encode all non-ASCII characters.
-                jsonSerializerOptions = jsonSerializerOptions.Copy(JavaScriptEncoder.UnsafeRelaxedJsonEscaping);
+                jsonSerializerOptions = new JsonSerializerOptions(jsonSerializerOptions)
+                {
+                    Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping,
+                };
```

Comment:
Hopefully the new `CreateForWeb` option uses the right defaults and then we don't need this at all 👀

Reply:
@rynowak @pranavkm It seems like you want us to include this in the new `JsonSerializerOptions` for web. So, the web options will have `PropertyNameCaseInsensitive = true`, `PropertyNamingPolicy = JsonNamingPolicy.CamelCase`, and `Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping`.

Is that correct?

I also pointed this out on https://github.com/dotnet/runtime/issues/34626#issuecomment-617595522 in case you missed it.
cc @layomia

Reply:
Also cc @terrajobst.

Reply:
AFAIK, when I spoke to @GrabYourPitchforks, the recommendation was that using  `UnsafeRelaxedJsonEscaping` is only safe if the JSON wasn't interspersed with markup. `Web` might not convey that this is meant for REST \ API scenarios.